packages/indicators/src/builders/buildArithmetic/aggregationConfig.ts

```diff
@@ -0,0 +1,145 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import { getVariables } from '@beyondessential/arithmetic';
+
+import { toArray } from '@tupaia/utils';
+import { ArithmeticConfig, AggregationDescriptor, AggregationSpecs } from './types';
+
+enum AggregationType {
```

Comment:
Here I used the Typescript convention for [enums](https://www.typescriptlang.org/docs/handbook/enums.html):

* Singular, not plural word
* PascalCase for the enum and its members

I also considered using our existing convention for JavaScript "enum" constants:

* Plural word
* upper SNAKE_CASE

I'm still not sure about what's the best approach. I like the TypeScript convention, but consistency is key.